src/Types/index.tsx

```diff
@@ -159,6 +159,11 @@ export enum DropdownItemType {
   None = 'none',
 }
 
+export enum IconPlacement {
```

Comment:
I think we have some duplication happening here. `IconPlacement` and `Direction` are essentially just `left` and `right`, and `Alignment` is `left`, `right`, and `center`. Perhaps we can consolidate these.

Also, `Orientation` and `ComponentOrientation` are just `horizontal` and `vertical`.

Reply:
got rid of icon placmeent for direction !